Library/Homebrew/extend/os/linux/requirements/osxfuse_requirement.rb

```diff
@@ -0,0 +1,11 @@
+require "requirement"
+
+class OsxfuseRequirement < Requirement
+  default_formula "libfuse"
```

Comment:
> I've made the assumption that if you're using brew and it's NOT on a Mac, you're using Linuxbrew's homebrew-core. Is that a reasonable assumption to make?

I think that's not a safe assumption yet so let's just omit `default_formula` for now.